packages/workspace/src/command-line/workspace-schematic.ts

```diff
@@ -139,12 +209,27 @@ function createWorkflow(dryRun: boolean) {
   });
 }
 
-function listSchematics(collectionName: string, logger: logging.Logger) {
+function listSchematics(
+  collection: ReturnType<typeof getCollection>,
+  logger: logging.Logger
+) {
   try {
-    const engineHost = new NodeModulesEngineHost();
-    const engine = new SchematicEngine(engineHost);
-    const collection = engine.createCollection(collectionName);
-    logger.info(engine.listSchematicNames(collection).join('\n'));
+    const bodyLines: string[] = [];
+
+    bodyLines.push(terminal.bold(terminal.green('WORKSPACE SCHEMATICS')));
```

Comment:
We use helper functions from output.ts to format everything else. Maybe we use them here as well?

Reply:
I used output.ts where it was easy-to replace, some capabilities needs proper unification around codebase. I'd suggest to address those in another PR.

> for example
> - I just copied `printHelp` from `tao/src/command/generate.ts` which still uses deprecate angular-devkit terminal API
> - `listPluginCapabilities` from workspace/plugins uses bodyLines array "approach" etc